lib/Sema/TypeCheckAttr.cpp

```diff
@@ -5915,9 +5915,41 @@ void AttributeChecker::visitDistributedActorAttr(DistributedActorAttr *attr) {
 }
 
 void AttributeChecker::visitDistributedKnownToBeLocalAttr(DistributedKnownToBeLocalAttr *attr) {
-  if (!D->isImplicit()) {
+  if (D->isImplicit()) {
+    // Implicit code is fine, this is how we use '_local' in synthesized thunks.
+    return;
```

Comment:
Hmm... I think that means we'll have to change synthesis of thunks to call `whenLocal`, because the thunks do "if local ... call the real func" (without using the whenLocal -- at least today), and they do so from `nonisolated` func -- so we'll have to change thunk synthesis to use whenLocal to get the `_local self` in order to do this 🤔 

That probably makes sense though, so thunks should become to use the normal function from here I suppose 🤔